internal/api/connectors/sources/mock.go

```diff
@@ -0,0 +1,42 @@
+package sources
+
+import (
+	"context"
+	"fmt"
+)
+
+var (
+	rhcId           string = "d415fc2d-9700-4e30-9621-6a410ccc92d8"
```

Comment:
Nit-picky:  You might consider making it so that you can inject these values into the mock, then you could remove these globals.